src/shared/sam/activation.ts

```diff
@@ -88,7 +88,7 @@ async function registerServerlessCommands(ctx: ExtContext): Promise<void> {
                 )
         ),
         vscode.commands.registerCommand('aws.lambda.createNewSamApp', async () => {
-            await createNewSamApplication(ctx.chanLogger, ctx.awsContext, ctx.regionProvider)
+            await createNewSamApplication(ctx)
```

Comment:
thanks for taking the time to make this change, instead of adding yet another param :)